src/modules/Channel/components/ChannelUI/index.tsx

```diff
@@ -122,9 +122,10 @@ const ChannelUI: React.FC<ChannelUIProps> = ({
         }
         <div className="sendbird-conversation__footer__typing-indicator">
           {
-            renderTypingIndicator?.() || (
-              <TypingIndicator />
-            )
+            renderTypingIndicator?.()
```

Comment:
It seems this file is only used for GroupChannel type but let me know if there's any chance that this can be used for OpenChannel too. Will add more condition then.

Reply:
yeah, is in groupChannel only for now ~